### PR TITLE
ETA bugfixes

### LINKED ISSUE #1667: ETA going backwards for unfinished abandoned buildings
https://youtu.be/gogr-q0nUMA

### PR DESCRIPTION
**Pull request by hihoman23**
### Work done
1. The ETA will now show `CANCELED` in red text and the decaying time will now show correctly for decaying buildings that haven't been fully built and don't have a constructor.
2. The ETA is moved down a bit to make it more visible under the construction progress bar. 

Closes #1667

### CODE REVIEW

File: luaui/Widgets/gui_build_eta.lua [hunk 1]
```diff
@@ -175,6 +175,8 @@ local function drawEtaText(timeLeft, yoffset)
 	if timeLeft == nil then
 		etaText = etaPrefix .. "\255\1\1\255???"
 	else
+		etaPrefix = ((timeLeft>0) and etaPrefix) or " \255\255\1\1CANCELED "
```
Comment (WatchTheFort) [inline luaui/Widgets/gui_build_eta.lua:178]: All text displayed to the player needs to use the I18N module, to support translations.

**Comment by WatchTheFort** [comment]
Is this the best way to indicate building decay? Could this be indicated by changing the timer to red and having it blink?

File: luaui/Widgets/gui_build_eta.lua (hunk 1 above)
Comment (hihoman23) [reply luaui/Widgets/gui_build_eta.lua:178]: Canceled is American, Cancelled is British English

**Comment by hihoman23** [comment]
> Is this the best way to indicate building decay? Could this be indicated by changing the timer to red and having it blink?

I think that showing the `CANCELED` and blinking might be the best option, but I'm not sure

**Comment by hihoman23** [comment]
@WatchTheFort Is this what you mean?

https://github.com/beyond-all-reason/Beyond-All-Reason/assets/78002940/355982f4-08b9-42ec-88e6-37ca47d746dc

**Comment by thehobojoe** [comment]
I think "decaying" might be a better term, "canceled" sounds like it's about to get deleted or something, when in reality the decay time is very long, even for small cheap structures.

**Comment by hihoman23** [comment]
Hmm, I couldn't see it yet, but this is what you mean, right? https://github.com/beyond-all-reason/Beyond-All-Reason/pull/2815/commits/ea6ebb92478c907b5f0ea2ca5c7982ee42ccbdb0

**Pull request merged**